docs/user/apirule-migration/01-84-migrate-oauth2-v1beta1-to-v2.md

```diff
@@ -0,0 +1,221 @@
+# Migrate APIRule `v1beta1` of type oauth2_introspection to version `v2`
+
+Learn how to migrate an APIRule created in version `v1beta1` using the **oauth2_introspection** handler to the **extAuth** handler in version `v2`. In APIRule `v2`, the **extAuth** handler replaces all Ory Oathkeeper-based handlers used in the `v1beta1` version. The tutorial focuses on **oauth2_introspection** as it is the most popular Ory Oathkeeper-based handler.
```

Comment:
```suggestion
Learn how to migrate an APIRule created in version `v1beta1` using the **oauth2_introspection** handler to the **extAuth** handler in version `v2`. In APIRule `v2`, the **extAuth** handler replaces all Ory Oathkeeper-based handlers used in the `v1beta1` version. The instructions focus on **oauth2_introspection** because it is the most popular Ory Oathkeeper-based handler.
```
I tend to avoid the word "tutorial" because in SAP/Kyma context, that can mean several different things :D

Reply:
true :D